pkg/vulnsrc/ghsa/ghsa.go

```diff
@@ -136,9 +137,11 @@ func (vs VulnSrc) commit(tx *bolt.Tx, ecosystem types.Ecosystem, entries []Entry
 		}
 
 		pkgName := vulnerability.NormalizePkgName(ecosystem, entry.Package.Name)
-		err = vs.dbc.PutAdvisoryDetail(tx, vulnID, pkgName, []string{bucketName}, a)
-		if err != nil {
-			return xerrors.Errorf("failed to save GHSA: %w", err)
+		if ecosystem != vulnerability.Go { // we only keep vulnerability details for Go
```

Comment:
We should mention we use `golang/vulndb` instead of GHSA and the reason why we don't use GHSA.
https://github.com/golang/vulndb

Reply:
Done